src/ActiveLogin.Authentication.BankId.AspNetCore/Models/IBankIdQrCodeGenerator.cs

```diff
@@ -0,0 +1,7 @@
+namespace ActiveLogin.Authentication.BankId.AspNetCore.Models
+{
+    public interface IBankIdQrCodeGenerator
```

Comment:
Please write a small "intellisense documentation" with one line of what it does.

Reply:
ok!